cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -615,12 +620,40 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         return declStmt
     }
 
+    private fun addExternallyImportedToAnalysis(importName: Name) {
+        ctx?.let { ctx ->
```

Comment:
I would suggest checking ctx for null and then throw a translation exception instead of having this nested.